unit/src/MFEMPostprocessorTest.C

```diff
@@ -51,10 +51,36 @@ TEST_F(MFEMPostprocessorTest, MFEML2Error)
   _scalar_var->ProjectCoefficient(twos);
   EXPECT_GT(l2_pp.getValue(), 1.);
 
-  _scalar_var->ProjectCoefficient(*_mfem_problem->getScalarFunctionCoefficient("scalar_ones"));
+  _scalar_var->ProjectCoefficient(
+      _mfem_problem->getCoefficients().getScalarCoefficient("scalar_ones"));
   EXPECT_LT(l2_pp.getValue(), 1e-12);
 }
 
+/**
+ * Test a corresponding coefficient has been created for MFEML2Error.
+ */
+TEST_F(MFEMPostprocessorTest, MFEML2ErrorCoefficient)
+{
+  InputParameters pp_params = _factory.getValidParams("MFEML2Error");
+  pp_params.set<FunctionName>("function") = "scalar_ones";
+  pp_params.set<VariableName>("variable") = "scalar_var";
```

Comment:
Seems like this postprocessor should be updated to the coefficient/functor system 😄